mne/time_frequency/psd.py

```diff
@@ -113,15 +113,19 @@ def _check_nfft(n, n_fft, n_overlap):
 
 
 @verbose
-def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, tmin=None,
-                       tmax=None, n_fft=256, n_overlap=0, proj=False,
-                       n_jobs=1, verbose=None):
+def compute_epochs_psd(epochs, picks=None, Fs=None, fmin=0, fmax=np.inf,
+                       tmin=None, tmax=None, n_fft=256, n_overlap=0,
+                       proj=False, method='welch',  mt_bandwidth=None,
+                       mt_adaptive=False, mt_low_bias=True,
+                       mt_normalization='length', n_jobs=1, verbose=None):
     """Compute power spectral density with average periodograms.
 
     Parameters
     ----------
-    epochs : instance of Epochs
-        The epochs.
+    epochs : instance of Epochs, array with epochs shape
+        The epochs. If not an Epochs object, it must be a numpy array
+        with shape (n_epochs, n_channels, n_times). Sampling frequency must
```

Comment:
Sampling frequency `Fs`

(otherwise the user will be wondering which argument is the sampling frequency or if this argument must also include the sampling frequency)